cmd/prchecker/main.go

```diff
@@ -41,79 +71,163 @@ func main() {
 
 	client := github.NewClient(nil)
 
-	files, err := ListAllPullRequestFiles(client, owner, pullrequest, repo)
+	githubRepo := &GitHubRepository{
```

Comment:
Nit: Since this is just a wrapper around basic types, it doesn't need to be a pointer.

Reply:
Disregard this comment. Now that the struct is implementing an interface, it makes sense to use a pointer here.